schema/bom-1.6.proto

```diff
@@ -71,6 +71,8 @@ enum Classification {
   CLASSIFICATION_MACHINE_LEARNING_MODEL = 11;
   // A collection of discrete values that convey information.
   CLASSIFICATION_DATA = 12;
+  // Cryptographic asset including algorithms, protocols, certificates, keys, tokens, and secrets.
+  CLASSIFICATION_CRYPTOGRAPHIC_ASSET = 13;
```

Comment:
Can I also request? This would help narrow down the search space while looking for crypto-related operations in large codebases. It is also fine as-is and could be considered for 1.7.

CLASSIFICATION_CRYPTOGRAPHIC_FRAMEWORK
CLASSIFICATION_CRYPTOGRAPHIC_LIBRARY

Reply:
- what makes a CLASSIFICATION_CRYPTOGRAPHIC_FRAMEWORK different from a CLASSIFICATION_FRAMEWORK?
what makes a CLASSIFICATION_CRYPTOGRAPHIC_LIBRARY different from a CLASSIFICATION_LIBRARY?

Reply:
If a framework or library is only used for building crypto-related functionality. For example, bouncycastle, openssl, etc.

Reply:
what would be a use case of such information? 
please elaborate the reasoning behind your request.

Reply:
To generate a cbom, if we know which libraries offer crypto-related functionalities, we can then get selective with the analysis.

Reply:
please create a dedicated issue for that, in case it is still needed :-)